docker/conda_build.dockerfile

```diff
@@ -0,0 +1,98 @@
+ARG ubuntu_version=22.04
+
+FROM ubuntu:${ubuntu_version} AS openmc
+
+# Ubuntu Setup
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+ENV HOME /root
+RUN apt-get update \
+    && apt-get install -y --fix-missing \
+        wget \
+        bzip2 \
+        ca-certificates \
+    && apt-get clean -y
+
+RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
+    wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
+    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh
+
+ENV PATH /opt/conda/bin:$PATH
+
+# install python 3.10 because that's what apt uses
+RUN conda update conda
+RUN conda install "python=3.10"
+RUN conda config --add channels conda-forge
+RUN conda update -n base -c defaults conda
+RUN conda install -y conda-libmamba-solver
+RUN conda config --set solver libmamba
+RUN conda install -y mamba
+RUN conda uninstall -y conda-libmamba-solver
+RUN conda config --set solver classic
+RUN conda update -y --all && \
+    mamba install -y \
+                gxx_linux-64 \
+                gcc_linux-64 \
+                cmake \
+                make \
+                gfortran \
+                libblas \
+                liblapack \
+                eigen \
+                numpy \
+                scipy \
+                nose \
+                matplotlib \
+                git \
+                setuptools \
+                pytest \
+                pytables \
+                jinja2 \
+                "cython<3" \
```

Comment:
```suggestion
                cython \
```

Reply:
Good catch - I think we're all updated for Cytron 3 now, right?